src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BlazorDesignTimeNodeWriter.cs

```diff
@@ -468,6 +499,27 @@ IReadOnlyList<IntermediateToken> GetCSharpTokens(ComponentAttributeExtensionNode
             }
         }
 
+        public override void WriteTemplate(CodeRenderingContext context, TemplateIntermediateNode node)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            if (node == null)
+            {
+                throw new ArgumentNullException(nameof(node));
+            }
+
+            // Since the user doesn't write this name themselves, we have to use something hardcoded
+            // and predicatable.
+            const string VariableName = "item";
```

Comment:
Thoughts on this vs context?

Reply:
Hmm how about ```DataContext```? Not too sure but It would probably help out those coming from the XAML world since it uses a lot of Templates for user controls there.